public/components/map_container/maps_messages.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiLink } from '@elastic/eui';
+import React from 'react';
+import { ToastInputFields } from '../../../../../src/core/public';
+import { toMountPoint } from '../../../../../src/plugins/opensearch_dashboards_react/public';
+
+export const RegionConflictWarningMsg: ToastInputFields = {
+  title: 'The default Dashboards Maps Service is currently not available in your region',
```

Comment:
```suggestion
  title: 'The OpenSearch Maps Service is currently not available in your region',
```

Reply:
ack

Reply:
@naveentatikonda There should be a period at the end of this notification. Also, the hyperlinked text could be limited to just the "custom map" portion of text instead of the entire sentence. Otherwise looks good!

Reply:
@carolxob Thanks for reviewing it. I have addressed those changes.